### DISCUSSION TITLE
*** buffer overflow detected ***: terminated

### DISCUSSION THREAD

**Discussion by ZdendaH** [OpenRadioss/OpenRadioss#625] [Q&A]
Hello OpenRadioss,

I got this message when I run the ice_cube and tensile_test examples:

 ***** buffer overflow detected ***: terminated** 

Maybe it can be caused by computer memory, I only have 16GB. 

Thank you for your answer.

**Comment by PaulAltair** [comment]
16GB is plenty, the ice cube run is only using a few MB of memory, This seems to be a bug in the h3d writer, thank you for reporting, we will investigate. In the engine file (0001.rad) you can switch to the Anim type output instead (comment out or delete the /H3D entries and uncomment the /ANIM ones... ) The tensile test doesn't have commented out /ANIM entries already, but you can do similar there (I attached an example for the first tensile test model)

```
/VERS/2019
/RUN/cube_TYPE7/1/
2.00000000000000    
#/H3D/NODA/CONT
#/H3D/NODA/VEL
#/H3D/SOLID/TENS/STRESS
#/H3D/COMPRESS
#0.01
#/H3D/DT
#0.000000000000000    0.04000000000000   
/RFILE
100000
#
/ANIM/VECT/CONT
/ANIM/VECT/VEL
/ANIM/BRICK/TENS/STRESS/000       
/ANIM/ELEM/EPSP
/ANIM/MASS
/ANIM/DT
0.000000000000000    0.04000000000000   
/PRINT/-500/0         
/TFILE/0         
0.005000000000000   
/DT
0.900000000000000    0.000000000000000   


```
[cube_TYPE7_0001.rad.txt](https://github.com/OpenRadioss/OpenRadioss/files/9791825/cube_TYPE7_0001.rad.txt)
[tensile_LAW2_0001.rad.txt](https://github.com/OpenRadioss/OpenRadioss/files/9791836/tensile_LAW2_0001.rad.txt)

**Comment by OlivierWienholtz** [comment]
Hello,
I created an issue of this discution, as there is likely a Bug causing this.
